index.php

```diff
@@ -8,12 +8,14 @@
 require_once 'autoload.php';
 
 use Mautic\CoreBundle\ErrorHandler\ErrorHandler;
+use Mautic\CoreBundle\Loader\EnvironmentHandler;
 use Mautic\Middleware\MiddlewareBuilder;
 use Symfony\Component\HttpFoundation\Request;
 
-ErrorHandler::register('prod');
+$config = (new EnvironmentHandler())->getEnvParameters();
+ErrorHandler::register($config['ENV']);
 
-$kernel   = (new MiddlewareBuilder(new AppKernel('prod', false)))->resolve();
+$kernel   = (new MiddlewareBuilder(new AppKernel($config['ENV'], $config['DEBUG'])))->resolve();
```

Comment:
```suggestion
$kernel   = (new MiddlewareBuilder(new AppKernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG'])))->resolve();
```
See https://github.com/symfony/recipes/blob/main/symfony/framework-bundle/4.4/public/index.php#L23